godotproject/Level0.tscn

```diff
@@ -132,19 +121,19 @@ bus = "Music"
 
 [node name="TextTip1" parent="." instance=ExtResource( 17 )]
 position = Vector2( 296.648, 885.363 )
-text = "[center] press enter/space/z to  [wave amp=35]jump"
+text = "[center] Press Left Button to  [wave amp=35]jump[/wave] and Right Button to [wave amp=35] shoot[/wave]"
 
 [node name="TextTip2" parent="." instance=ExtResource( 17 )]
 position = Vector2( 806.767, 629.771 )
-text = "[center] jump and launch tongue at a block to [wave]swing [/wave] using shift/x"
+text = "[center] jump and launch tongue at a block to [wave]swing [/wave] "
 
 [node name="TextTip3" parent="." instance=ExtResource( 17 )]
 position = Vector2( 6828, 453.701 )
-text = "[center] press shift/x while swinging to [wave] launch [/wave] yourself up"
+text = "[center] press Right Button while swinging to [wave] launch [/wave] yourself up"
 
 [node name="TextTip4" parent="." instance=ExtResource( 17 )]
 position = Vector2( 2587.93, 661.872 )
-text = "[center] you can control the direction of your tongue: try shooting [wave] diagonally"
+text = "[center] You can control the direction of your tongue: try shooting [wave] diagonally"
```

Comment:
I think we need to align on the capitals here - it seems kinda inconsistent to have some things start with lowercase and some with uppercase

Reply:
Good call, I'll go through and make sure its consistent